usr/share/rear/backup/NETFS/default/41_save_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# save all found capapilities to file 
+
+if [ "$BACKUP_CAP" == "y" ] ; then
+	Log "Saving capabilities to File." 
+	if which getcap >/dev/null 2>&1 ; then
+		getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities
```

Comment:
Yes, the `getcap` tool is flawed and not made for automation. Are there other tools for this task?

Also, what happens if you record capability of a file that is not part of the actual backup? Will `setcap` fail? Or do we silently ignore errors when calling `setcap` to restore the capabilities?

In general, you need to have a clean base line if you want to be able to handle actual errors. If you don't have a clean baseline then you won't be able to distinguish real errors (needing human intervention) from "normal" errors which are safe to ignore.